src/js/resize-manager.js

```diff
@@ -109,6 +109,7 @@ class ResizeManager extends Component {
   }
 
   dispose() {
+    this.trigger('dispose');
```

Comment:
should we call `super.dispose()` here instead?

Reply:
yeah that probably makes more sense.